addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11432,7 +11432,30 @@ msgctxt "#21330"
 msgid "Show hidden files and directories"
 msgstr ""
 
-#empty strings from id 21331 to 21335
+#: xbmc/storage/AutorunMediaJob.cpp
```

Comment:
still strings could be reused by devs in future and thus a contextual comment would help. It also helps translators to know if a string is f.e. a dialog headline, message text or a settings value (f.e. to adjust length of translation)